deployment/helm/values.yaml

```diff
@@ -287,4 +287,52 @@ udev:
     # targetPort is the service targetPort of the instance service
     targetPort: 8083
     # protocol is the service protocol of the instance service
-    protocol: TCP
\ No newline at end of file
+    protocol: TCP
+
+coapcore:
+  # enabled defines whether to load a coapcore configuration
+  enabled: false
+  # name is the Kubernetes resource name that will be created for this
+  # onvif configuration
+  name: coap-core
+  # properties is a map of properties that will be passed to any instances
+  # created as a result of applying this onvif configuration
+  properties:
+  ipAddresses: []
+  # capacity is the capacity for any instances created as a result of
+  # applying this onvif configuration
+  capacity: 1
+  brokerPod:
+    image:
+      # repository is the container reference
+      repository:
+      # pullPolicy is the Akri onvif broker pull policy
+      pullPolicy: Always
+  # createInstanceServices is specified if a service should automatically be
+  # created for each broker pod
+  createInstanceServices: true
+  instanceService:
+    # name is the description of the instance service
+    name: coap-core-instance-service
+    # type is the service type of the instance service
+    type: ClusterIP
+    # port is the service port of the instance service
+    port: 80
+    # targetPort is the service targetPort of the instance service
+    targetPort: 8083
+    # protocol is the service protocol of the instance service
+    protocol: TCP
+  # createConfigurationService is specified if a single service should automatically be
+  # created for all broker pods of a Configuration
+  createConfigurationService: true
```

Comment:
This probably should be `false` by default, doesn't make a lot of sense to have a service of services in CoAP I think.

Reply:
it seems like that would depend on how the broker is structuring the data it is sending.  if it just sends a number without context (72, 34), then i would agree.  but if there is some context (temp=72, humidity=34), then it seems fine to have config service.  i haven't fully wrapped my head around what this protocol means yet, but if a single config might discover many instances that are all the same type (5 instances for 5 thermometers), then the config service would allow someone to view all of them.

Reply:
We are discussing what a config means for CoAP. I'll leave as it is now then, for the time being.